recipe/meta.yaml

```diff
@@ -1,48 +1,53 @@
 {% set name = "qpdf" %}
-{% set version = "10.6.3" %}
+{% set version = "11.9.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/qpdf/qpdf/archive/release-qpdf-{{ version }}.tar.gz
-  sha256: 64bbb654ec19b6812284ef1bc40c57b091de5281f265f1c3c83dd8ccf6bedcf5
+  url: https://github.com/qpdf/qpdf/releases/download/v{{ version }}/qpdf-{{ version }}.tar.gz
+  sha256: 9f5d6335bb7292cc24a7194d281fc77be2bbf86873e8807b85aeccfbff66082f
 
 build:
-  number: 2
-  skip: true  # [win]
+  number: 0
 
 requirements:
   build:
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-    - make
-    - automake
-    - autoconf
-    - coreutils  # [osx]
+    - {{ stdlib("c") }}
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - cmake
+    - ninja
   host:
-    - zlib
     - libjpeg-turbo
-  run:
+    - openssl
```

Comment:
This supports OpenSSL or GnuTLS. Given how commonly available OpenSSL is (including on Windows) and how much usage it gets in conda-forge, stuck with OpenSSL

Though wanted to note this in case there are other thoughts